src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg"
+        alt="imac" 
+        class="card__image"
+      >
+
+      <h1 class="card__info">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
```

Comment:
better to use h2, cause in the next task you will have a lot of product hards on the page, but it only can be one h1 element on one page

```suggestion
      <h2 class="card__info">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h1>
```

Reply:
also add font-weight to this text according to the design, now it differs from it
![image](https://user-images.githubusercontent.com/57598825/152780347-9c4a2291-97a1-4cb5-b434-932559837952.png)

Reply:
в фигме font-weight указан 500, я у себя поставил 500... Я либо фигмой не умею пользоваться, либо не знаю :)) 
![image](https://user-images.githubusercontent.com/98328152/152782533-648c6c42-e3df-4d92-aedd-2984e3428fba.png)
![image](https://user-images.githubusercontent.com/98328152/152782552-9271e809-cbcd-43fc-8eef-2d0597c4f9e7.png)